qp_klp/klp.py

```diff
@@ -158,3 +161,138 @@ def sequence_processing_pipeline(qclient, job_id, parameters, out_dir):
+
+    sheet = MetagenomicSampleSheetv90()
+    sheet.Header['IEMFileVersion'] = '4'
+    sheet.Header['Date'] = datetime.today().strftime('%m/%d/%y')
+    sheet.Header['Workflow'] = 'GenerateFASTQ'
+    sheet.Header['Application'] = 'FASTQ Only'
+    sheet.Header['Assay'] = prep_info['data_type']
+    sheet.Header['Description'] = f'prep_NuQCJob - {pid}'
+    sheet.Header['Chemistry'] = 'Default'
+    sheet.Header['SheetType'] = 'standard_metag'
+    sheet.Header['SheetVersion'] = '90'
+    sheet.Header['Investigator Name'] = 'Qiita'
+    sheet.Header['Experiment Name'] = project_name
+
+    sheet.Bioinformatics = pd.DataFrame(
+        columns=['Sample_Project', 'ForwardAdapter', 'ReverseAdapter',
+                 'library_construction_protocol',
+                 'experiment_design_description',
+                 'PolyGTrimming', 'HumanFiltering', 'QiitaID'],
+        data=[[project_name, 'NA', 'NA', 'NA', 'NA', 'FALSE', 'TRUE', sid]])
+
+    for k, vals in pt.iterrows():
+        k = k.split('.', 1)[-1]
+        sample = {
+            'Sample_Name': k,
+            'Sample_ID': k.replace('.', '_'),
+            'Sample_Plate': '',
+            'well_id_384': '',
+            'I7_Index_ID': '',
+            'index': vals['index'],
+            'I5_Index_ID': '',
+            'index2': vals['index2'],
+            'Sample_Project': project_name,
+            'Well_description': '',
+            'Sample_Well': '',
+            'Lane': '1'}
+        sheet.add_sample(sample_sheet.Sample(sample))
+
+    sheet.Contact = pd.DataFrame(
+        columns=['Email', 'Sample_Project'],
+        data=[['qiita.help@gmail.com', project_name]])
+
+    new_sample_sheet = out_path('sample-sheet.csv')
+    with open(new_sample_sheet, 'w') as f:
+        sheet.write(f, 1)
+
+    # now that we have a sample_sheet we can fake the
+    # ConvertJob folder so we are ready for the restart
+    convert_path = out_path('ConvertJob')
+    project_folder = out_path('ConvertJob', project_name)
+    makedirs(project_folder, exist_ok=True)
+
+    for _, fs in files.items():
+        for f in fs:
+            bn = basename(f['filepath'])
+            symlink(f['filepath'], f'{project_folder}/{bn}')
+
+    # create job_completed file to skip this step
+    with open(f'{convert_path}/job_completed', 'w') as f:
+        f.write('')
+
+    try:
+        kwargs = {'qclient': qclient,
+                  'uif_path': new_sample_sheet,
+                  'lane_number': "1",
+                  'config_fp': CONFIG_FP,
+                  'run_identifier': '211021_A00000_0000_SAMPLE',
+                  'output_dir': out_dir,
+                  'job_id': job_id,
+                  'status_update_callback': status_line.update_job_status,
+                  # set 'update_qiita' to False to avoid updating Qiita DB
+                  # and copying files into uploads dir. Useful for testing.
+                  'update_qiita': True,
+                  'is_restart': True}
```

Comment:
```suggestion
    kwargs = {'qclient': qclient,
              'uif_path': new_sample_sheet,
              'lane_number': "1",
              'config_fp': CONFIG_FP,
              'run_identifier': '211021_A00000_0000_SAMPLE',
              'output_dir': out_dir,
              'job_id': job_id,
              'status_update_callback': status_line.update_job_status,
              # set 'update_qiita' to False to avoid updating Qiita DB
              # and copying files into uploads dir. Useful for testing.
              'update_qiita': True,
              'is_restart': True}
    try:
```